.github/workflows/netsim.yml

```diff
@@ -97,7 +97,7 @@ jobs:
     - name: Copy binaries to right location
       run: |
         cp target/optimized-release/iroh ../chuck/netsim/bins/iroh
-        cp target/optimized-release/derper ../chuck/netsim/bins/derper
+        cp target/optimized-release/iroh-relay ../chuck/netsim/bins/iroh-relay
```

Comment:
To cheat for now you can do `cp target/optimized-release/iroh-relay ../chuck/netsim/bins/derper` given commands/args have remained the same. I can do another pass to bring all the other infra in line with the rename.